<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>471</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="search_line"/>
           </item>
           <item>
            <widget class="QPushButton" name="search_button">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string> pid           state    mem        cpu            name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="widget_list">
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="stop_button">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="continue_button">
           <property name="text">
            <string>Continue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="kill_button">
           <property name="text">
            <string>Kill</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>451</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>                                         SYSTEM INFORMATION</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Hostname:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hostname_lable">
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>User:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="user_lable">
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>CPU:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cpu_lable">
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>RAM:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ram_lable">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>451</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>                                                 PERFOMANCE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>CPU:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="cpu_progress">
           <property name="minimumSize">
            <size>
             <width>204</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>204</width>
             <height>28</height>
            </size>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Memory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="memory_progress">
           <property name="minimumSize">
            <size>
             <width>204</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>204</width>
             <height>28</height>
            </size>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Uptime:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="uptime_lable">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionTasks"/>
    <addaction name="actionPerfomance"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuSort_by">
     <property name="title">
      <string>Sort by</string>
     </property>
     <addaction name="actionName"/>
     <addaction name="actionPid"/>
     <addaction name="actionPercent_CPU"/>
     <addaction name="actionPercwnt_memory"/>
    </widget>
    <widget class="QMenu" name="menuSearch_by">
     <property name="title">
      <string>Search by</string>
     </property>
     <addaction name="actionName_2"/>
     <addaction name="actionPid_2"/>
    </widget>
    <addaction name="menuSort_by"/>
    <addaction name="menuSearch_by"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <widget class="QMenu" name="menuAbout_us">
    <property name="title">
     <string>About us</string>
    </property>
    <addaction name="actionAbout_QT"/>
    <addaction name="actionAbout_programm"/>
    <addaction name="actionAbout_author"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout_us"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTasks">
   <property name="text">
    <string>Tasks</string>
   </property>
  </action>
  <action name="actionPerfomance">
   <property name="text">
    <string>Perfomance</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionPid">
   <property name="text">
    <string>Pid</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionPriority">
   <property name="text">
    <string>Priority</string>
   </property>
  </action>
  <action name="actionPercent_CPU">
   <property name="text">
    <string>Percent CPU</string>
   </property>
  </action>
  <action name="actionPercwnt_memory">
   <property name="text">
    <string>Percent memory</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionName_2">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionPid_2">
   <property name="text">
    <string>Pid</string>
   </property>
  </action>
  <action name="actionPriority_2">
   <property name="text">
    <string>Priority</string>
   </property>
  </action>
  <action name="actionUser_2">
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionAbout_programm">
   <property name="text">
    <string>About programm</string>
   </property>
  </action>
  <action name="actionAbout_author">
   <property name="text">
    <string>About author</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
